container/containerd/handler.go

```diff
@@ -112,11 +116,22 @@ func newContainerdContainerHandler(
 		rootfs = "/rootfs"
 	}
 
+	aliases := []string{id, name}
+
+	if *useK8sContainerNameFormat && cntr.Labels["io.cri-containerd.kind"] == "container" {
+		restart := cntr.Labels["io.kubernetes.pod.restart"]
```

Comment:
add note that this label is not populated and possibly TODO linking to issue to track this